src/Mapbender/CoreBundle/Component/Application.php

```diff
@@ -332,7 +333,8 @@ public function getConfiguration()
             $layerSets     = array();
 
             foreach ($layerSet->layerObjects as $layer) {
-                $instHandler = EntityHandler::createHandler($this->container, $layer);
+                /** @var SourceInstanceEntityHandler|WmsInstanceEntityHandler $instHandler */
+                $instHandler = SourceInstanceEntityHandler::createHandler($this->container, $layer);
```

Comment:
I would prefer to have here an injected service to increase decoupling, then call of a static method of a class, if possible

Reply:
This branch is pretty old! Config service has already been implemented. This whole section is gone after the merge commit.